src/coord/src/catalog/builtin.rs

```diff
@@ -774,6 +797,250 @@ pub const PG_ENUM: BuiltinView = BuiltinView {
     needs_logs: false,
 };
 
+// The following types are the list of builtin data types available
+// in Materialize. This list is derived from the Type variants supported
+// in pgrepr.
+//
+// Builtin types cannot be created, updated, or deleted. Their OIDs
+// are static, unlike other objects, to match the type OIDs defined by Postgres.
+pub const TYPE_BOOL: BuiltinType = BuiltinType {
```

Comment:
I don't have `List` here because I'm not sure how that's translated to a Postgres type.